src/applications/travel-pay/redux/reducer.js

```diff
@@ -164,6 +156,7 @@ const initialState = {
 };
 
 function travelPayReducer(state = initialState, action) {
+  /* eslint-disable sonarjs/max-switch-cases */
```

Comment:
I'm not seeing this lint error locally when removing this lint disable. Is this something you get lint errors for?